pkgs/top-level/all-packages.nix

```diff
@@ -18498,7 +18498,7 @@ with pkgs;
     })
     subversion18 subversion19 subversion_1_10;
 
-  subversion = pkgs.subversion19;
```

Comment:
Looking at the changelog it also seems that `subversion19` is not an LTS release, so we probably want to remove it before the next release.

Reply:
[Here](http://subversion.apache.org/download.cgi#supported-releases) it seems that 1.9 will be supported until 1.11 is released. I think we should keep 1.9, but remove 1.8.